certora/specs/SocialRecoveryModule.spec

```diff
@@ -436,3 +436,25 @@ rule canFinalizeRecoveryOnlyAfterDelayPeriod(env e) {
 
     assert success => require_uint64(e.block.timestamp) >= recoveryTimestamp, "Recovery finalized before delay period";
 }
+
+// Recovery can be finalized by anyone after the delay period is over.
+rule anyoneCanFinalizeRecovery(env e) {
+    uint64 executeAfter = currentContract.recoveryRequests[safeContract].executeAfter;
+
+    requireInitiatedRecovery(safeContract);
+
+    require e.msg.value == 0;
```

Comment:
This can be removed. The prover will automatically learn that `msg.value` must be 0 for the call to succeed.

Reply:
As we are checking both conditions (success and failure), we need to ensure this to avoid any other failing reason other than `delayPeriod` not being over.